provider/cluster/kube/builder.go

```diff
@@ -62,7 +62,7 @@ func (b *nsBuilder) create() (*corev1.Namespace, error) {
 	}, nil
 }
 
-func (b *nsBuilder) update(obj *corev1.Namespace) (*corev1.Namespace, error) {
+func (b *nsBuilder) update(obj *corev1.Namespace) (*corev1.Namespace, error) { // nolint
```

Comment:
maybe suppress a particular linter instead of all?

Reply:
Thanks for the  idea, I didn't know that was a feature! Implemented.